packages/api/test/unit/keymanager/testData.ts

```diff
@@ -13,6 +13,7 @@ const pubkeyRand = "0x84105a985058fc8740a48bf1ede9d223ef09e8c6b1735ba0a55cf4a9ff
 const ethaddressRand = "0xabcf8e0d4e9587369b2301d0790347320302cc09";
 const graffitiRandUtf8 = "636861696e736166652f6c6f64657374";
 const gasLimitRand = 30_000_000;
+const builderBoostFactorRand = BigInt(100);
```

Comment:
There's a bunch of new hardcoded `100`, could a constant be used more globally?

Reply:
also added to todo cleanup list